addons/skin.estuary/xml/SettingsSystemInfo.xml

```diff
@@ -84,15 +84,15 @@
 				<left>420</left>
 				<right>50</right>
 				<top>100</top>
-				<bottom>315</bottom>
+				<bottom>340</bottom>
```

Comment:
```suggestion
				<bottom>350</bottom>
```

@ronie sorry for bothering you again ;)
this change would prevent the privacy policy textbox from being glued to the line underneath:

![glued](https://user-images.githubusercontent.com/58829855/83797749-61dd6200-a6a3-11ea-9390-28cd285d18f2.png)

on top there's a gap:

![unglued](https://user-images.githubusercontent.com/58829855/83798024-dd3f1380-a6a3-11ea-89a1-c1cb0d387a4a.png)

...apart from that thumbs up

Reply:
> @ronie sorry for bothering you again ;)
> this change would prevent the privacy policy textbox from being glued to the line underneath:
> 
> ![glued](https://user-images.githubusercontent.com/58829855/83797749-61dd6200-a6a3-11ea-9390-28cd285d18f2.png)

people who take the time and effort to test my pr's can bother me as much as they want ;-)

though i'm unsure what problem you're referring to here...
the screenshot looks ok to me?

> on top there's a gap:
> 
> ![unglued](https://user-images.githubusercontent.com/58829855/83798024-dd3f1380-a6a3-11ea-89a1-c1cb0d387a4a.png)

true that. but it's also like that in the current version of estuary.
will fix that as well.

Reply:
yeah, the gap at top of the textbox is correct. if the text starts scrolling you can see that the gap at the _bottom_ is missing

Reply:
ok, i've re-added the gaps. both at the top & bottom.